apps/vr-tests-react-components/src/stories/Link.stories.tsx

```diff
@@ -2,10 +2,29 @@ import * as React from 'react';
 import { Steps, StoryWright } from 'storywright';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 24 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28341&commitId=50f6023b2e81033808f8cec10c5af6eaf503b435&env=prod)

✅ There was 24 screenshots added, 0 screenshots removed, 1954 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as anchor.On inverted background - Dark Mode.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background - Dark Mode.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background - Dark Mode.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background - Dark Mode.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background - High Contrast.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background - High Contrast.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background - High Contrast.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background - High Contrast.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.On inverted background.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background - Dark Mode.default.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background - Dark Mode.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background - Dark Mode.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background - Dark Mode.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background - High Contrast.default.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background - High Contrast.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background - High Contrast.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background - High Contrast.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background.default.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.On inverted background.pressed.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
@ling1726 it looks like these screenshots aren't getting cropped to the content. Is it related to the `InvertedBackground` wrapper?

Reply:
Or.. it looks like all of the link stories might be missing the TestWrapperDecorator (if that's the case, then it could be fixed in a separate PR).